src/mon/AuthMonitor.cc

```diff
@@ -1687,7 +1687,7 @@ bool AuthMonitor::_upgrade_format_to_dumpling()
     // set daemon profiles
     if ((p->first.is_osd() || p->first.is_mds()) &&
         mon_caps == "allow rwx") {
-      new_caps = string("allow profile ") + p->first.get_type_name();
+      new_caps = string("allow profile ") + std::string(p->first.get_type_name());
```

Comment:
Tiny nit: as string() has already been used, specifying std::string isn't needed-- and /also/ you can probably get away with p->first.get_type_name() and std::string::operator+() probably will do the right thing. Note that libfmt is also now available!

Reply:
string + string_view seems not to be defined.